x11-themes/numix-gtk-theme/numix-gtk-theme-2.6.7.ebuild

```diff
@@ -18,4 +18,5 @@ RDEPEND="dev-libs/libxml2
 DEPEND="${RDEPEND}
 	dev-libs/glib:2
 	dev-ruby/sass:*
+	!=dev-ruby/sass-3.4.25-r1
```

Comment:
Weak blockers are used to solve file collisions. They don't guarantee that sass-3.4.25-r1 will be unmerged prior to installing this.

Reply:
Yeah, I was thinking of doing the 
```
|| ( 
<dev-ruby/sass-3.4.25-r1
>dev-ruby/sass-3.4.25-r1
)
```
solution, whatever correct syntax might be but you get the idea. In the end however to my eyes this `!...` looked better. It doesn't need to be uninstalled, as sass-3.4.25-r1 can co-exist with sass:3.5 (it's slotted). I think the whole idea of 3.4.25-r1 is so its libraries can co-exist with :3.5 slot of sass, since the only colliding parts are the binaries. 

I tested this solution and it does pull another version of sass if 3.4.25-r1 is the only installed one, but I can switch it to a solution mentioned above, too. I was hoping ruby team would do something about it, since this whole situation is a bit weird, but the -r1 bump seems intentional so there must be a good reason this "broken" package still exists in Gentoo. :I

Reply:
I gave it a thought and maybe blocker isn't the best choice, since 3.4.25-r1 can be installed IF other version from another slot is present in the system too. 

I'm gonna switch this into `|| ( <dev-ruby/sass-3.4.25-r1 >dev-ruby/sass-3.4.25-r1 )`. I'll test the logic to make sure it works, and update this PR if it does.

Reply:
Maybe make it `>=dev-ruby/sass-3.5`, in case -r2 was ever added with same problem. Also, `>=` should go first as the preferable solution.

Reply:
Good point. Yeah I thought about adding `>=dev-ruby/sass-3.5` as dep too, it's a shame that previous versions still do work though. However I have no intention of asking this to be stabilized (where 3.4 is needed) so I'll switch to that and commit update soonish.